extensions/src/main/java/org/pentaho/platform/web/http/api/resources/FileResource.java

```diff
@@ -2480,7 +2525,7 @@ protected String getUserHomeFolder() {
   /**
    * Checks if the given users and roles are valid, i.e. don't contain illegal characters.
    * Illegal characters may lead to repository corruption.
-   *
```

Comment:
**Not a blocking PR merge PR comment.**

was there any substantive code change for `FileResource.java` or was it all formatting? Can you add these changes in a separate commit  with [CLEANUP].

Reply:
The change that was made is in the "chore" commit already. The description in that commit includes a statement for what was done to this class:
```
- Removed unused field of UserRoleListService from FileResource
```

It was important to remove the unused field because some breaking changes were made to `UserRoleListService`. This emphasizes that there is no code besides the `UserRoleListResource` class is using the service, thus validating that the breaking changes are safe to be made.